sdk/python/feast/infra/offline_stores/remote.py

```diff
@@ -175,8 +223,31 @@ def write_logged_features(
         logging_config: LoggingConfig,
         registry: BaseRegistry,
     ):
-        # TODO Implementation here.
-        raise NotImplementedError
```

Comment:
do we need to update the end to end example to call these function using client as done for get_historical_features 
 https://github.com/RHEcosystemAppEng/feast/tree/remote_offline/examples/remote-offline-store

https://github.com/RHEcosystemAppEng/feast/blob/remote_offline/examples/remote-offline-store/offline_client/test.py#L29

Reply:
Sounds good if you want to update you can but, since its not an actual test not sure how much will it contribute